pyproject.toml

```diff
@@ -1,5 +1,49 @@
+[tool.poetry]
+name = "R2T2"
+version = "0.3.1"
+description = "Research references tracking tool"
+repository = "https://github.com/ImperialCollegeLondon/R2T2"
+homepage = "https://imperialcollegelondon.github.io/R2T2/"
+authors = ["Research Computing Service, Imperial College London <rcs-support@imperial.ac.uk>"]
+license = "MIT"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: MIT License",
+    "Topic :: Utilities",
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+wrapt = "^1.12.0"
+
+[tool.poetry.dev-dependencies]
```

Comment:
Should this list not also include black? (note: I think you might need to allow pre-releases if you want black and poetry to play nice)

Reply:
Maybe? I only included things that were already listed as dependencies or otherwise clearly implied because they were used in the CI workflow. There's configuration for black and isort if you choose to use them, but it's developer's choice.